tuning/sft_trainer.py

```diff
@@ -286,23 +296,67 @@ def main(**kwargs):
         choices=["pt", "lora", None, "none"],
         default="pt",
     )
+    parser.add_argument(
+        "--exp_metadata",
+        type=str,
+        default=None,
+    )
     (
         model_args,
         data_args,
         training_args,
         lora_config,
         prompt_tuning_config,
-        peft_method,
+        aim_config,
+        additional,
         _,
     ) = parser.parse_args_into_dataclasses(return_remaining_strings=True)
-    if peft_method.peft_method == "lora":
-        tune_config = lora_config
-    elif peft_method.peft_method == "pt":
-        tune_config = prompt_tuning_config
+
+    peft_method = additional.peft_method
+    if peft_method =="lora":
```

Comment:
you'll want to run black on your code, there should be a space after the `==`

Reply:
thanks for the pointer. done